source/agent/quic/webtransport/quicTransportServer.js

```diff
@@ -151,7 +151,9 @@ module.exports = class QuicTransportServer extends EventEmitter {
             }
             return;
           }
-        }
+        };
+        // In case data arrived before ondata handler is assigned.
+        stream.checkReadableData();
```

Comment:
I know the reason you why add this. 
But it's a little confusing when I look at this first time. (1. If the data arrived before, why not just ignore it? 2. If someone assigned ondata handler like 1 min later, will he still get readable data from beginning?)

If user must call `checkReadableData` right after `ondata` assignment, can you move the logic to a `setter` method for `ondata` in addon layer? (I've never used JS setter in addon, not sure whether it's feasible)

Reply:
A WebTransport stream is associated with a publication or a subscription. The first 16 bytes or 32 bytes cannot be ignored because they're publication/subscription ID and track ID. After reading 16 bytes or 32 bytes, the stream is piped to a sink based on their type (signaling, media or data). The sink decides how to handle data received. It may drop data if no receiver available.

`ondata` is a pure JavaScript object, not handled by addon, so addon doesn't know when it is assigned. It's a good idea to check readable data after `ondata` is set. Let me check whether we can implement getter and setter for `ondata` in addon.

Reply:
Implemented getter and setter. Checking readable stream was moved to setter.